xwiki-platform-core/xwiki-platform-blocknote/xwiki-platform-blocknote-webjar/src/main/node/package.json

```diff
@@ -9,21 +9,21 @@
     "test:unit": "vitest --run"
   },
   "dependencies": {
-    "@xwiki/cristal-api": "catalog:",
-    "@xwiki/cristal-attachments-default": "catalog:",
-    "@xwiki/cristal-authentication-api": "catalog:",
-    "@xwiki/cristal-backend-api": "catalog:",
-    "@xwiki/cristal-collaboration-api": "catalog:",
-    "@xwiki/cristal-document-api": "catalog:",
-    "@xwiki/cristal-dsapi": "catalog:",
-    "@xwiki/cristal-editors-blocknote-headless": "catalog:",
-    "@xwiki/cristal-editors-blocknote-react": "catalog:",
-    "@xwiki/cristal-link-suggest-api": "catalog:",
-    "@xwiki/cristal-model-api": "catalog:",
-    "@xwiki/cristal-model-reference-api": "catalog:",
-    "@xwiki/cristal-model-remote-url-api": "catalog:",
-    "@xwiki/cristal-uniast-api": "catalog:",
-    "@xwiki/cristal-uniast-markdown": "catalog:",
+    "@xwiki/platform-api": "workspace:*",
```

Comment:
The `@xwiki/cristal-*` packages used to be fetched from a remote registry and are now pulled from the current workspace. They are not part of the catalog anymore.